dev-java/bcmail/bcmail-1.77.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="org.bouncycastle:bcmail-jdk18on:${PV}"
+JAVA_TESTING_FRAMEWORKS="junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="The Bouncy Castle Java S/MIME APIs for handling S/MIME protocols"
+HOMEPAGE="https://www.bouncycastle.org/java.html"
+SRC_URI="https://github.com/bcgit/bc-java/archive/r${PV/./rv}.tar.gz -> bc-java-r${PV/./rv}.tar.gz"
+S="${WORKDIR}/bc-java-r${PV/./rv}/mail"
```

Comment:
`r${PV/./rv}` is used at several places in all the ebuilds. i think it would be cleaner to use `MY_PV="r${PV/./rv}"` and then `${MY_PV}` everywhere. it's much cleaner to reuse just the variable instead of coding the replacement everywhere. this affects all the ebuilds.